Robot_oM/Enums/CombinationType_EC.cs

```diff
@@ -0,0 +1,44 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2025, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+namespace BH.oM.Adapters.Robot
+{
+    /***************************************************/
+    /****               Public Enums                ****/
+    /***************************************************/
+
+    public enum CombinationType_EC
+    {
+        Default = 0,
+        ULS, // Ultimate Limit State
+        SLS, // Serviceability Limit State
```

Comment:
```suggestion
        EC_ULS, // Ultimate Limit State
        EC_SLS, // Serviceability Limit State
```
Renaming the `Enum` to `LoadCombinationType` gives you scope to add more for different design standards at a later stage.

I would also encourage the use of `[Description()]` and `[DisplayText()]` attributes, an example you can see here:
https://github.com/BHoM/BHoM/blob/8ea74db21b95e81f2901a87475b1d2b03e56be16/Ground_oM/eNums/Legend.cs#L33-L35